modules/features2d/include/opencv2/features2d.hpp

```diff
@@ -529,11 +531,16 @@ class CV_EXPORTS_W GFTTDetector : public Feature2D
     CV_WRAP virtual void setBlockSize(int blockSize) = 0;
     CV_WRAP virtual int getBlockSize() const = 0;
 
+    CV_WRAP virtual void setGradientSize(int gradientSize);
+    CV_WRAP virtual int getGradientSize();
+
     CV_WRAP virtual void setHarrisDetector(bool val) = 0;
     CV_WRAP virtual bool getHarrisDetector() const = 0;
 
     CV_WRAP virtual void setK(double k) = 0;
     CV_WRAP virtual double getK() const = 0;
+protected:
+    int gradientSize;
```

Comment:
This breaks ABI again.
You need to add this field to `GFTTDetector_Impl`, not to `GFTTDetector`